resources/metadata/abapEnvironmentCreateSystem.yaml

```diff
@@ -179,6 +179,46 @@ spec:
           - STAGES
           - STEPS
           - GENERAL
+      - name: abapSystemParentServiceLabel
+        description: The name of the parent service label that is passed through to CLD
+        type: string
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: abapSystemParentServiceInstanceGUID
+        description: The guid/Service ID of the parent service
+        type: string
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: abapSystemParentSaaSAppname
+        description: The parent appname from the SaaS registry
+        type: string
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: abapSystemParentServiceParameters
+        description: JSON string that contains the service parameters of the parent service passed through CLD
+        type: string
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: abapSystemConsumerTenantLimit
```

Comment:
We should discuss with Welf wether this parameter should already be included. During system creation it is available, but the functionality for it does not work yet.